plugins/vestige/vestige.cpp

```diff
@@ -263,7 +263,7 @@ void vestigeInstrument::loadFile( const QString & _file )
 	{
 		tf = TextFloat::displayMessage(
 				tr( "Loading plugin" ),
-				tr( "Please wait while loading VST-plugin..." ),
+				tr( "Please wait while loading VST plugin..." ),
```

Comment:
Maybe add a "the" before "VST"? In this case it's talking about a specific VST plugin that it's loading and not just any plugin.